app/src/main/java/com/brewfind/Service/BreweryService.kt

```diff
@@ -10,7 +10,7 @@ import retrofit2.Response
 
 class BreweryService {
     internal fun fetchBreweries() : MutableLiveData<ArrayList<Brewery>> {
-        var breweries = MutableLiveData<ArrayList<Brewery>>()
+        var _breweries = MutableLiveData<ArrayList<Brewery>>()
```

Comment:
Changing breweries to _breweries according to coding conventions.
Source: [https://kotlinlang.org/docs/coding-conventions.html#names-for-backing-properties](https://kotlinlang.org/docs/coding-conventions.html#names-for-backing-properties)

Reply:
This isn't a backing property, so the _ convention doesn't apply.